hw/top_earlgrey/sw/autogen/top_earlgrey.h

```diff
@@ -2,8 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef OPENTITAN_HW_TOP_EARLGREY_SW_AUTOGEN_TOP_EARLGREY_H_
-#define OPENTITAN_HW_TOP_EARLGREY_SW_AUTOGEN_TOP_EARLGREY_H_
+#pragma once
```

Comment:
Ok. At the moment, the script is broken. I'll change this PR so that it just fixes the script and leaves the behaviour the same, disallowing `#pragma once`.

As a result, other engineers are going to have to fix things up in C++ all over the tree because they violate this particular corner of the style guide (which is now going to be checked in CI). Other corners aren't automatically checked, and are violated in much of the DV / verilator interaction code. If you feel strongly that they should also follow the Google style guide, you're going to have to file lots of PRs.

Reply:
> If you feel strongly that they should also follow the Google style guide, you're going to have to file lots of PRs.

Yeah, we should be following the style guide. This should be something we can absolutely do mechanically and if we want to turn this on in CI outside of sw/ we need to write tooling for it. I don't have the cycles to write this tooling but I can't allow this to leak into sw/.

Reply:
@rswarbrick do you mean the OpenTitan style guide, which is a derivative of the Google style guide? It's worth stating for the avoidance of doubt, that yes, style guides must be followed for commits on this project. If there are problems, raise issues. 

If there are multiple violations in the codebase, either they or the style guide need to be fixed. @asb is there a technical issue here we need to look into?

Reply:
#5693 and #5702 should fix all in-tree code to match the preferred include guard style. I think these files just landed before we had the automated checking for this put in place.